hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/clustering/HoodieFlinkClusteringJob.java

```diff
@@ -62,6 +64,8 @@ public class HoodieFlinkClusteringJob {
 
   protected static final Logger LOG = LoggerFactory.getLogger(HoodieFlinkClusteringJob.class);
 
+  private static final String NO_EXECUTE_CALL_KEYWORD = "The application contains no execute() calls";
+
```

Comment:
We can simplified the keyword as `no execute` instead.

Reply:
Done